src/sentry/integrations/github/integration.py

```diff
@@ -164,34 +164,53 @@ def get_client(self) -> GitHubBaseClient:
             raise IntegrationError("Organization Integration does not exist")
         return GitHubApiClient(integration=self.model, org_integration_id=self.org_integration.id)
 
+    # IntegrationInstallation methods
+
     def is_rate_limited_error(self, exc: Exception) -> bool:
         if exc.json and RATE_LIMITED_MESSAGE in exc.json.get("message", ""):
             metrics.incr("github.link_all_repos.rate_limited_error")
             return True
 
         return False
 
-    def get_trees_for_org(self, cache_seconds: int = 3600 * 24) -> dict[str, RepoTree]:
-        trees: dict[str, RepoTree] = {}
-        domain_name = self.model.metadata["domain_name"]
-        extra = {"metadata": self.model.metadata}
-        if domain_name.find("github.com/") == -1:
-            logger.warning("We currently only support github.com domains.", extra=extra)
-            return trees
+    def message_from_error(self, exc: Exception) -> str:
+        if not isinstance(exc, ApiError):
+            return ERR_INTERNAL
 
```

Comment:
I wonder if this sort of error handling/extraction is something we could also define a new method for in the ABC

Reply:
this is from `IntegrationInstallation`, we can make it an `@abstractmethod` but it also has a default implementation